include/eal.h

```diff
@@ -0,0 +1,21 @@
+#ifndef EAL_H
+#define EAL_H
+
+#include <vector>
+#include <string.h>
+
+using namespace std;
+
+vector<double> getPosition(vector<vector<double>> coords, vector<double> coeffs, char *interp, double time);
+vector<double> getVelocity(vector<vector<double>> coords, vector<double> coeffs, char *interp, double time, bool interpolation);
+
+vector<double> getPosition(vector<double> coeffs, char *interp, double time);
+vector<double> getVelocity(vector<double> coeffs, char *interp, double time, bool interpolation);
```

Comment:
We should pass the interpolation as an enumeration instead of a c-string. This will help avoid string parsing and allow us to use a switch statement in the implementation if we want to.